src/phx/api/phx_api.py

```diff
@@ -172,7 +173,10 @@ def dispatch(self):
                 # call internal handler
                 match msg:
                     case OrderBookUpdate():
-                        self.on_order_book_update(msg)
+                        if msg.symbol in self.set_of_symbol_names:
+                            self.on_order_book_update(msg)
+                        else:
+                            continue
```

Comment:
please add debug message here (even temporary) to verify we get into this statement?

Reply:
I just ran an integrated test with debug messages to verify this!